openmls/src/group/mls_group/membership.rs

```diff
@@ -248,14 +248,14 @@ impl MlsGroup {
         self.plaintext_to_mls_message(remove_proposal, backend)
     }
 
-    /// Gets the current list of members
-    pub fn members(&self) -> Result<Vec<&Credential>, MlsGroupError> {
+    /// Gets a list of [`KeyPackage`]s of the current group members.
+    pub fn members(&self) -> Result<Vec<&KeyPackage>, MlsGroupError> {
```

Comment:
Can we get a test for this.
There are tests using this that could be extended.
Some scenarios how this could be used
- check that some member is some sender, i.e. check that the key package reference matches a certain value
- check a member's credential (this might be covered already)
- address members in the group, e.g. to remove

Reply:
Good point, I'll extend the tests as you describe.